manifests/ha/base/kustomization.yaml

```diff
@@ -21,3 +14,10 @@ images:
 - name: argoproj/argocd-ui
   newName: argoproj/argocd-ui
   newTag: latest
+resources:
+- ../../base/application-controller
+- ../../base/dex
+- ../../base/repo-server
+- ../../base/server
+- ../../base/config
+- ./redis-ha
```

Comment:
I don't think we should force people to use v2.1.0 just yet. Don't think theres a v2.1 feature we are leveraging to warrant the potential disruption.